web/client/epics/search.js

```diff
@@ -182,9 +182,13 @@ const zoomAndAddPointEpic = action$ =>
                     id: "searchPoints",
                     visibility: true,
                     style: {
-                        iconGlyph: "comment",
-                        iconShape: "circle",
-                        iconColor: "green"
+                        color: '#3388ff',
+                        weight: 4,
+                        dashArray: '',
+                        fillColor: '#3388ff',
+                        fillOpacity: 0.2,
+                        iconUrl: "https://cdn.rawgit.com/pointhi/leaflet-color-markers/master/img/marker-icon-red.png",
```

Comment:
can we use a local copy of this one?

Reply:
yes, i was thinking that if one overrides the resultsStyle of the Search plugin this style will be different. i'll check if we can find it from the state, (my guess is that from the config is passed to the plugin but not in the state :) )

Reply:
My comment was about not depending from an external url that could disappear.

Reply:
i know, that's why i started with a yes... 🥇